getting-started/scaling-out.md

```diff
@@ -0,0 +1,172 @@
+# Scaling out using Distributed Hypertables
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
+Hypertables are used to handle large amount of data by breaking it up
+into smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+Data nodes together with an *access node* constitue a multi-node cluster
```

Comment:
constitute (typo. Should probably run spell check on document)

Reply:
Done